src/redux/Categories/categorySlice.js

```diff
@@ -1,14 +1,14 @@
 import { createReducer } from '@reduxjs/toolkit';
```

Comment:
According to the requirement, you should be importing a method called createSlice instead of createReducer this method helps you to do things way easier. Please include it to keep your code consistent with the requirements. Read about [createSlice](https://redux-toolkit.js.org/api/createSlice) here